core/settings.py

```diff
@@ -150,7 +151,7 @@ def get_env_variable(name, cast=str, default=""):
             "django_browser_reload",
         ]
     )
-    SHELL_PLUS_PRINT_SQL = True
+    SHELL_PLUS_PRINT_SQL = get_env_variable("SHELL_PLUS_PRINT_SQL", cast=bool)
```

Comment:
```suggestion
    SHELL_PLUS_PRINT_SQL = get_env_variable("SHELL_PLUS_PRINT_SQL", default=True, cast=bool)
```

Please 🙏 comme ça je n'ai pas besoin de changer quoi que ce soit en local 🙉

Reply:
ah ah si tu veux